types/proposal_test.go

```diff
@@ -96,28 +91,46 @@ func TestProposalVerifySignature(t *testing.T) {
 }
 
 func BenchmarkProposalWriteSignBytes(b *testing.B) {
+	pbp := getTestProposal(b).ToProto()
+
+	b.ResetTimer()
+
```

Comment:
I think using a `b.Run` subbenchmark may be a bit more clear for communicating what the intention of only timing the code that occurs after the test proposal is created.

Reply:
that's fair. I was just trying to clean up the current state of the tests wrt setup and what not without being too heavy handed. ResetTimer is _the way_ to do this, and a single subtest also seems a bit unclear in other ways.